components/tools/CategoryDropdown.js

```diff
@@ -0,0 +1,17 @@
+import ToolsData from '../../config/tools.json';
+
+export default function CategoryDropdown({setopenCategory}) {
+
+  return (
+  <div class="absolute z-10 w-52 lg:w-56 origin-top-right h-60 overflow-y-auto rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="menu-button" tabindex="-1">
```

Comment:
```suggestion
  <div class="absolute z-10 -left-[2rem] lg:left-0 w-52 lg:w-56 origin-top-right h-60 overflow-y-auto rounded-md bg-white shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none" role="menu" aria-orientation="vertical" aria-labelledby="menu-button" tabindex="-1">
```

To remove the scroll from smaller screens.

Reply:
Yeah, I found this solution to be not working correctly on different screens. I've another way of doing it and it is in the suggestions below.